tardis/io/tests/data/tardis_configv1_verysimple.yml

```diff
@@ -5,7 +5,6 @@ supernova:
     time_explosion: 13 day
 
 atom_data: kurucz_atom_pure_simple.h5
```

Comment:
this is fine - if as you say this is never really used we could put a dummy string in. but leave it for now.